core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.js

```diff
@@ -47,7 +47,7 @@ module.exports = class baseUser {
         /** TODO(#17761): Right now some acceptance tests are failing on
          * headless mode. As per the expected behavior we need to make sure
          * every test passes on both modes. */
-        headless: false,
+        headless: true,
```

Comment:
No, The test doesn't work if headless is false so I have to change it to true. #17761 was requesting to fix it so tests can pass in true and false.

Reply:
It seems that other existing acceptance tests work with headless=false -- at least, they pass on CI. Why doesn't this test work with headless=false? (Asking because the file you're editing here is a utility file that affects all acceptance tests, right?)

Reply:
I agree with @seanlip. This change will affect all tests, so we need to pick one mode (headless=true or headless=false) and make sure that all the tests pass in that mode

Reply:
On my pc, if headless=false, an error saying "Missing X server or $DISPLAY" will pop up therefore I changed it to true. But since other existing acceptance tests work with headless=false then I'm guessing it's a problem in the setting on my pc. I will go ahead and fix the comment and push the commit with headless=false and see if it works.

Reply:
Done